<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLogin</class>
 <widget class="QDialog" name="UserLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/img/8552951.png</normaloff>:/img/8552951.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15, 163, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="LBsing">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sing in to coutinue</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLineEdit" name="LEquestionLogin">
     <property name="minimumSize">
      <size>
       <width>204</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 2px solid rgb(0, 0, 0);
}


QLineEdit:hover {
    color: #000000;
    border: 2px solid rgb(0, 0, 0);
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>enter a system question...</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QComboBox" name="CBloginAs">
     <property name="minimumSize">
      <size>
       <width>204</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 2px solid rgb(0, 0, 0);
}</string>
     </property>
     <item>
      <property name="text">
       <string>Login as:</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>користувач</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>адміністратор</string>
      </property>
     </item>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLineEdit" name="LEloginU">
     <property name="minimumSize">
      <size>
       <width>204</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 2px solid rgb(0, 0, 0);
}


QLineEdit:hover {
    color: #000000;
    border: 2px solid rgb(0, 0, 0);
}</string>
     </property>
     <property name="placeholderText">
      <string>enter login...</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLineEdit" name="LEpasswordU">
     <property name="minimumSize">
      <size>
       <width>204</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 2px solid rgb(0, 0, 0);
}


QLineEdit:hover {
    color: #000000;
    border: 2px solid rgb(0, 0, 0);
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>enter password...</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="PBnextU">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 3px solid rgb(0, 0, 0);
}


QPushButton:hover {
    color: #000000;
    background-color: rgb(201, 201, 201); 
    border: 2px solid rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>next</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="PBbackU">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    color: #000000;
    background-color: rgb(219, 219, 219);
    border-radius: 7px; 
    border: 3px solid rgb(0, 0, 0);
}


QPushButton:hover {
    color: #000000;
    background-color: rgb(201, 201, 201); 
    border: 2px solid rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
